ocetrac/model.py

```diff
@@ -44,8 +47,16 @@ def __init__(self, da, mask, radius, min_size_quartile, xdim, ydim):
         self.mask = mask
         self.radius = radius
         self.min_size_quartile = min_size_quartile
+        self.timedim = timedim
         self.xdim = xdim
         self.ydim = ydim   
+        
+        if ((timedim, ydim, xdim) != da.dims):
```

Comment:
Dont you have to wrap this in `set(...)` to order it?

Reply:
Nvm, i misunderstood the code.

Reply:
Done!
```python 
assert ((timedim, ydim, xdim) != Anom.expand_dims('level').dims)
````